packages/back-end/src/python/test/test_dists.py

```diff
@@ -0,0 +1,143 @@
+from functools import partial
+from unittest import TestCase, main as unittest_main
+
+import numpy as np
+import pandas as pd
+from scipy.special import digamma
+from scipy.stats import beta, norm
+from dists import Beta, Norm
```

Comment:
I'm not super familiar with relative importing in Python.  When I try to run the tests (`python3 test/test_dists.py`), I'm getting an error `ModuleNotFoundError: No module named 'dists'`.

I want to integrate these tests into the existing CI pipeline.  Is there something I need to run first to make this work?

Reply:
i'll check it out
does test main work for you?

Reply:
I've changed the sources root to `python` directory. tell me if that work for you

Reply:
I was able to get it to work by adding `__init__.py` in both the `test` and `bayesian` directories, turning them into modules. I also needed to tweak how the python script is being called from the application.  I committed those changes and integrated the python unit tests to the CI/CD pipeline.

Need to fix one more bug with the CI/CD pipeline and then I think it's good to merge.

Reply:
@itamarfaran Everything looks good to me now.  Is this ready to merge?

Reply:
ok from my end